go-controller/pkg/ovn/master.go

```diff
@@ -410,7 +418,7 @@ func (oc *DefaultNetworkController) addNode(node *kapi.Node) ([]*net.IPNet, erro
 	// subsequent operation in addNode() fails, oc.lsManager.DeleteNode(node.Name)
 	// needs to be done, otherwise, this node's IPAM will be overwritten and the
 	// same IP could be allocated to multiple Pods scheduled on this node.
-	err = oc.ensureNodeLogicalNetwork(node, hostSubnets)
+	err = oc.createNodeLogicalSwitch(node.Name, hostSubnets, oc.clusterLoadBalancerGroupUUID, oc.switchLoadBalancerGroupUUID)
```

Comment:
the reason why you had to split out createNodeLogicalSwitch is because we now need to add address-sets for local and remote nodes right?

Reply:
Will revert this change and force push